--- intel-driver-1.0.18/configure.ac.bero	2013-01-16 03:32:46.131157433 +0100
+++ intel-driver-1.0.18/configure.ac	2013-01-16 03:32:50.896100419 +0100
@@ -21,7 +21,7 @@ AC_INIT([intel_driver], [intel_driver_ve
 AC_CONFIG_SRCDIR([Makefile.am])
 AM_INIT_AUTOMAKE
 
-AM_CONFIG_HEADER([src/config.h])
+AC_CONFIG_HEADERS([src/config.h])
 
 INTEL_DRIVER_MAJOR_VERSION=intel_driver_major_version
 INTEL_DRIVER_MINOR_VERSION=intel_driver_minor_version
